examples/auth-generate-token/page.md

```diff
@@ -0,0 +1,151 @@
+# Generate a JWT token to authenticate clients
+
+Use JWT tokens to securely authenticate users within your application.
+
+JWT tokens enable you to grant users access to Ably's real-time messaging services. They are most commonly used to issue time-limited credentials that securely authorize users to connect to Ably channels.
+
+JWT tokens are an essential component of applications that require secure and scalable real-time communication. Before any application can use any of the Ably products whether they be Pub/Sub, Spaces, Chat, or LiveSync, they'll need to first authenticate.
+
+Authentication is implemented using [Ably core Pub/Sub](https://ably.com/docs/auth). The Pub/Sub SDK contains a set of purpose-built APIs that abstract away the complexities involved in architecting websocket features. It is built on top of Ably's core platform, and so it provides the same performance guarantees and scaling potential.
```

Comment:
I think we need to reword this to state that authentication is handled by Pub/Sub and how that bleeds into the other products.